hplsql/src/main/java/org/apache/hive/hplsql/ColumnMap.java

```diff
@@ -28,13 +28,25 @@ public class ColumnMap {
   private List<Column> columns = new ArrayList<>();
   private Map<String, Column> columnMap = new HashMap<>();
 
+  private Map<String, String> columnNameVsFQColumnName = new HashMap<>();
```

Comment:
IMHO we should store the fully qualified name in an object which has two String fields. One for table/alias and another for column name